src/components/Groups/Groups.js

```diff
@@ -0,0 +1,46 @@
+import {Box1} from "../Events/EventsContainer";
+import UserGroups, {CardHeader} from "./UserGroups";
+import {Container} from "@material-ui/core";
+import {styled} from "@material-ui/styles";
+import { connect } from "react-redux";
+import Group from "./Group";
+
+
+const GroupsPage= styled(Container)({
+    display: `flex`,
+    justifyContent: `space-between`,
+    padding: `4rem 0`,
+    backgroundColor: `#f7fdfc`
+})
+
+const AllGroups = styled(Box1)({
+    padding: `1rem`,
+    minWidth: `50vw`,
+    backgroundColor: `#ebfaf7`
+})
+
+const GroupsContainer = (props) => {
+    const filtered = props.groups.filter(
+        (group) => !props.userGroups.joined.map((group) => group.groupId).includes(group.groupId))
```

Comment:
Would it be better to take out the !props.userGroups.joined.map((group) => group.groupId) portion as a const above so like:

const userGroupIds = props.userGroups.joined.map((userGroup) => userGroup.groupId)
const filtered = props.groups.filter((group) => !userGroupIds.includes(group.groupId))

It's kinda of convoluted to have it all on one line and hard to read. The filter and map function passes in a parameter that are named the same, so its even harder to dissect. Also you call the map function on the same props.userGroups every time you iterate through with the filter function, so you can pull it out for inconsequential performance reasons.